### PR TITLE
afe:ad4110:Function additions, struct update

### PR DESCRIPTION
**Pull request by jansunil**
Added new functions to configure the device
Updated the init structure and device descriptor
Corrected the bit shift value for AD4110_REG_ADC_CONFIG_AIN_BUFF

Signed-off-by: Janani Sunil <Janani.Sunil@analog.com>

### CODE REVIEW

File: drivers/afe/ad4110/ad4110.c [hunk 1]
```diff
@@ -830,6 +966,26 @@ int32_t ad4110_setup(struct ad4110_dev **device,
 	if (ret)
 		goto err_spi;
 
+	ret = ad4110_set_bipolar(dev, init_param.bipolar);
+	if (ret)
+		goto err_spi;
+	dev->bipolar = init_param.bipolar;
```
Comment (amiclaus) [inline drivers/afe/ad4110/ad4110.c:972]: why not making the assignment `dev->bipolar=init_param.bipolar;` inside the setter function if the write is completed successfully?
Same applies for all the other setter functions.

In this manner the users can benefit of these functions outside the driver and the devices structure is updated accordingly.

Comment (jansunil) [reply drivers/afe/ad4110/ad4110.c]: Moved the device descriptor update inside the setter function.

**Comment by jansunil** [comment]
Following changed as a part of addressing review comments:
 - Moved the device desc update inside the setter function.

**Comment by jansunil** [comment]
Change made:
Added missing function prototype for ad4110_wait_for_rdy_low() in the ad4110.h file

**Comment by jansunil** [comment]
No changes made in the previous push. Edited the commit message and force pushed to re-trigger a build.

**Review by amiclaus — APPROVED** [review]

**Review by buha — APPROVED** [review]

**Pull request merged**